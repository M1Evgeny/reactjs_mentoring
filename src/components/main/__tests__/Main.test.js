import React from "react";
import { render } from "@testing-library/react";
import { Main } from "..";

describe("Main test", () => {
  test("Main snapshot", () => {
    const { asFragment } = render(<Main />);
    expect(asFragment()).toMatchSnapshot();
  });
});