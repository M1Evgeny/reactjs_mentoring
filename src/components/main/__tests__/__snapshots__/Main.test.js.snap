// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main test Main snapshot 1`] = `
<DocumentFragment>
  <main
    class="main"
  />
</DocumentFragment>
`;
